python/dgl/heterograph.py

```diff
@@ -4701,6 +4701,46 @@ def int(self):
                               self._node_frames,
                               self._edge_frames)
 
+    def reverse(self, share_ndata=True, share_edata=False):
+        r""" Same as dgl.reverse_heterograph
```

Comment:
Let's replace `dgl.reverse_heterograph` by `:func:`dgl.reverse_heterograph`` for redirection to the doc of dgl.reverse_heterograph.